### PR TITLE
Use KAS kubeconfig for PowerVS CCM instead of external kubeconfig

### PR DESCRIPTION
**Pull request by dharaneeshvrd**
**What this PR does / why we need it**:

Fixes https://issues.redhat.com/browse/MULTIARCH-3238

**Which issue(s) this PR fixes** *(optional, use `fixes #<issue_number>(, fixes #<issue_number>, ...)` format, where issue_number might be a GitHub issue, or a Jira story*:
Fixes #

**Checklist**
- [x] Subject and description added to both, commit and PR.
- [ ] Relevant issues have been referenced.
- [ ] This change includes docs. 
- [ ] This change includes unit tests.

### CODE REVIEW

File: control-plane-operator/controllers/hostedcontrolplane/cloud/powervs/powervs.go [hunk 1]
```diff
@@ -174,7 +174,7 @@ func ReconcileCCMDeployment(deployment *appsv1.Deployment, hcp *hyperv1.HostedCo
 						Name: hcp.Status.KubeConfig.Name,
```
Comment (mkumatag) [inline control-plane-operator/controllers/hostedcontrolplane/cloud/powervs/powervs.go:174]: can you change the name here and the name in the VolumeMounts section as well?

**Review by mkumatag — CHANGES_REQUESTED** [review]

File: control-plane-operator/controllers/hostedcontrolplane/cloud/powervs/powervs.go (hunk 1 above)
Comment (dharaneeshvrd) [reply control-plane-operator/controllers/hostedcontrolplane/cloud/powervs/powervs.go]: Renamed in other places.

**Review by mkumatag — APPROVED** [review]
/lgtm

File: control-plane-operator/controllers/hostedcontrolplane/cloud/powervs/powervs.go [hunk 2]
```diff
@@ -155,7 +155,7 @@ func ReconcileCCMDeployment(deployment *appsv1.Deployment, hcp *hyperv1.HostedCo
 						},
 						VolumeMounts: []corev1.VolumeMount{
 							{
-								Name:      hcp.Status.KubeConfig.Name,
+								Name:      "service-network-admin-kubeconfig",
```
Comment (csrwng) [inline control-plane-operator/controllers/hostedcontrolplane/cloud/powervs/powervs.go:158]: nit: pls use manifests.KASServiceKubeconfigSecret("").Name
That makes it easier to identify the components that reference that kubeconfig

**Comment by csrwng** [comment]
Just one nit, otherwise lgtm

File: control-plane-operator/controllers/hostedcontrolplane/cloud/powervs/powervs.go (hunk 2 above)
Comment (dharaneeshvrd) [reply control-plane-operator/controllers/hostedcontrolplane/cloud/powervs/powervs.go]: thanks for pointing out, fixed. PTAL

**Pull request merged**